react/features/filmstrip/functions.native.js

```diff
@@ -90,15 +115,101 @@ export function getColumnCount(stateful: Object | Function) {
 }
 
 /**
- * Returns true if thumbnail reordering is enabled and false otherwise.
+ * Returns true if the filmstrip has a scroll and false otherwise.
  *
  * @param {Object} state - The redux state.
- * @returns {boolean} - True if thumbnail reordering is enabled and false otherwise.
+ * @returns {boolean} - True if the scroll is displayed and false otherwise.
  */
-export function isReorderingEnabled(state) {
-    const { testing = {} } = state['features/base/config'];
-    const enableThumbnailReordering = testing.enableThumbnailReordering ?? true;
+export function isFilmstripScollVisible(state) {
+    if (shouldDisplayTileView(state)) {
+        return state['features/filmstrip']?.tileViewDimensions?.hasScroll;
+    }
 
-    return enableThumbnailReordering;
+    const { aspectRatio, clientWidth, clientHeight, safeAreaInsets = {} } = state['features/base/responsive-ui'];
+    const isNarrowAspectRatio = aspectRatio === ASPECT_RATIO_NARROW;
+    const disableSelfView = shouldHideSelfView(state);
+    const localParticipant = Boolean(getLocalParticipant(state));
```

Comment:
Ditto, why do we need to check for this? There is no way to be in a meeting without a local participant.

Reply:
Ditto. It is the same situation as `getTileViewParticipantCount`. So from what I see we dispatch some actions to register sounds before we create the local participant. Maybe there are some other actions dispatched but that doesn't matter. IMHO this will trigger the subscribers to run the condition function and the local participant won't be available. Isn't that true?

Let me know if I'm missing something and this is not true!